examples/get_event_200_extra_fields.json

```diff
@@ -45,7 +45,8 @@
           "userAgent": "Mozilla/5.0 (Windows NT 6.1; Win64; x64) ...."
         },
         "confidence": {
-          "score": 0.97
+          "score": 0.97,
+          "revision": "v1.1"
```

Comment:
Just for consistency, can you also add this to `webhook.json`, and all 200 response JSON examples?

Reply:
In most cases that would not be present, it may be more confusing to have this field everywhere